src/optable.ts

```diff
@@ -660,7 +556,7 @@ optable[opcodes.RAISE_VARARGS] = function(f: Py_FrameObject, t:Thread) {
         case 0:
             // todo: re-raising
             do_raise(f, t, cause, exc);
```

Comment:
I'm confused. Why are you raising and then trying to catch the exception? Shouldn't you throw the exception on the thread object, e.g. `thread.throwException`, which will handle finding the appropriate stack frame to handle the exception?